### DISCUSSION TITLE
Stopping the container seems to take longer than necessary

### DISCUSSION THREAD

**Discussion by damianh** [nagyesta/lowkey-vault#805] [General]
Hi there, lovely tool!

Not sure if this can be classified as an issue or not, however it seems to take 10+ seconds to stop the container. As a test dummy, I would somewhat expect that startup and shutdown to be snappy.

`docker run --rm -d --name lowkey nagyesta/lowkey-vault:2.1.26-jammy` -> ~1 second
`docker stop lowkey` -> 10+ seconds

Cheers :)

**Comment by nagyesta** [comment]
Hi @damianh ,
thank you for the feedback! It is an unintended behaviour, so I have created a new issue based on it. Not sure what might cause it, but I was able to reproduce it on an M1 Mac, I assume you are using one too (hence the multiarch image). Will keep you updated on #806 .

Thanks again!

**Reply by damianh** [reply]
Thanks for validating and creating an issue.

My team is mixed windows x64 and Mac M1/2 hence the multiarch image usages.
